pumpkin-world/src/chunk/anvil.rs

```diff
@@ -161,3 +167,136 @@ impl ChunkReader for AnvilChunkReader {
         ChunkData::from_bytes(decompressed_chunk, *at).map_err(ChunkReadingError::ParsingError)
     }
 }
+
+impl ChunkWriter for AnvilChunkFormat {
+    fn write_chunk(
+        &self,
+        chunk: &ChunkData,
+        save_file: &SaveFile,
+        at: pumpkin_core::math::vector2::Vector2<i32>,
+    ) -> Result<(), super::ChunkWritingError> {
+        // TODO: update timestamp
+
+        let bytes = chunk.to_bytes();
+
+        let region = (
+            ((at.x as f32) / 32.0).floor() as i32,
+            ((at.z as f32) / 32.0).floor() as i32,
+        );
```

Comment:
We use this also for reading, would probably make sense to have it as a function